src/data_provider/src/ports/portLinuxWrapper.h

```diff
@@ -226,9 +226,9 @@ class LinuxPortWrapper final : public IPortWrapper
 
             return retVal;
         }
-        int32_t inode() const override
+        int64_t inode() const override
         {
-            return std::stoi(m_fields.at(INODE));
+            return static_cast<int64_t>(std::stol(m_fields.at(INODE)));
```

Comment:
can we use stoll here? since the long long is necessarily at least 8 bytes according to the standard.

Reply:
Yes! stoll function will be appropriate to avoid any kind of problem! thanks @Dwordcito :+1: